backend/Origam.Rule/RowSecurityStateBuilder.cs

```diff
@@ -0,0 +1,222 @@
+﻿using CSharpFunctionalExtensions;
+using DiffPlex;
+using Origam.DA;
+using Origam.Schema.EntityModel;
+using Origam.Service.Core;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Text;
+using Origam.Extensions;
+using Microsoft.SqlServer.Server;
+
+namespace Origam.Rule
+{
+    public class RowSecurityStateBuilder
+    {
+        private static readonly log4net.ILog log =
```

Comment:
Please update the logger documentation as these messages will now have a different logger. https://community.origam.com/t/logging-configuration-loggers/750

Reply:
Added

Reply:
The way the rest of the loggers are documented is that the error message is mentioned (including placeholders) and an explanation of the error follows (when and why would it happen). Please update the documentation in this fashion.

Reply:
Also it might good to specify since which ORIGAM version this logger is (will be) available.

Reply:
Logger documentation align according to requests.